ConsoleApp1/Program.cs

```diff
@@ -2,8 +2,169 @@
 {
 	internal class Program
 	{
+		public enum SortAlgorithmType
+		{
+			SelectionSort,
+			BubbleSort,
+			InsertionSort
+		}
+
+		public enum OrderBy
+		{
+			Asc,
+			Desc
+		}
+
 		static void Main(string[] args)
 		{
+			Random random = new Random();
+			int length1 = random.Next(5,10);
+			int[] myArr = new int[length1];
+
+			for (int i = 0; i < length1 - 1; i++)
+			{
+				myArr[i] = random.Next(100);
+			}
+			Console.WriteLine("RANDOM VALUES ARRAY");
+			foreach (var item in myArr)
+			{
+				if (myArr.Length - 1 == Array.LastIndexOf(myArr, item))
+				{
+					Console.Write($"{item} ");
+				}
+				else
+				{
+					Console.Write($"{item}, ");
+				}
+			}
+			Console.WriteLine();
+			Console.WriteLine();
+			Console.WriteLine("Sorted random values array by Asc");
+
+			Sort(ref myArr, SortAlgorithmType.InsertionSort, OrderBy.Asc);
+			foreach (var item in myArr)
+			{
+				if (myArr.Length - 1 == Array.LastIndexOf(myArr, item))
+				{
+					Console.Write($"{item} ");
+				}
+				else
+				{
+					Console.Write($"{item}, ");
+				}
+			}
+			Console.WriteLine();
+			Console.WriteLine();
+
+			Console.WriteLine("Sorted random values array by Desc");
+			Sort(ref myArr, SortAlgorithmType.SelectionSort, OrderBy.Desc);
+			for (int i = 0; i < myArr.Length; i++)
```

Comment:
What is the logic inside the loop ?

Reply:
The logic of ALL these loops in the MAIN is to "remove" the last comma symbol when presenting the data:
EG.
Without loop:
1, 2, 3, 4, 5,
With loop:
1, 2, 3, 4, 5
As you can see, the reduntand "Comma" symbol is removed from presentation